backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -80,14 +81,18 @@
 import org.hl7.fhir.r4.model.ServiceRequest.ServiceRequestStatus;
 import org.hl7.fhir.r4.model.Specimen;
 import org.hl7.fhir.r4.model.StringType;
+import org.springframework.boot.info.GitProperties;
 import org.springframework.data.util.Pair;
+import org.springframework.util.CollectionUtils;
 
 @Slf4j
 public class FhirConverter {
   private FhirConverter() {
     throw new IllegalStateException("Utility class");
   }
 
+  private static final String SIMPLE_REPORT_ORG_ID = "07640c5d-87cd-488b-9343-a226c5166539";
```

Comment:
So when we refer to org id in this context is to represent the application that is sending the bundle to report stream?

Reply:
Yes that's right! There is part of the HL7 message ([SFT](https://hl7-definition.caristix.com/v2/HL7v2.6/Segments/SFT)) that they need the software name for. The SimpleReport ORG only has a name and no other information.